frontend/src/less/components/searchbox.less

```diff
@@ -1,15 +1,44 @@
 #searchbox {
+    width: @width-searchbox;
```

Comment:
@amcatko maybe more general name? :) as the width corresponds also to search input width and would correspond to future elements similarly placed to the center